pages/iframe/introduction.mdx

```diff
@@ -0,0 +1,12 @@
+# Introduction
+
+The iframe allows NFT collections to take advantage of OpenSea's [animation_url](https://docs.opensea.io/docs/metadata-standards) metadata param. Basically you can
```

Comment:
I would remove language in this casual voice and just tryo to be as objective and direct as possible. Take out words like 'basically' and any filler words.